recipes/doublemetaphone/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "DoubleMetaphone" %}
+{% set version = "0.1" %}
+{% set md5 = "86e5ae4a4839321f60b9801837eb01c1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [win and linux]
```

Comment:
Is this just `osx`? If so, this should be `# [win or linux]` or better yet `# [not osx]`. Currently this `skip` is getting dropped.